documentation/book/proc-configuring-mirror-maker.adoc

```diff
@@ -0,0 +1,129 @@
+* Source and target Kafka clusters are available
+
+.Procedure
+
+. Edit the `spec` properties for the `KafkaMirrorMaker` resource.
++
+The properties you can configure are shown in this example configuration:
++
+[source,yaml,subs="+quotes,attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaMirrorMaker
+metadata:
+  name: my-mirror-maker
+spec:
+  # ...
+  replicas: 3 <1>
+  consumer:
+    bootstrapServers: my-source-cluster-kafka-bootstrap:9092 <2>
+    groupId: "my-group" <3>
+    numStreams: 2 <4>
+    offsetCommitInterval: 120000 <5>
+    tls: <6>
+    trustedCertificates:
+      - secretName: my-cluster-cluster-cert
+        certificate: ca.crt
+    authentication: <7>
+      type: tls
+      certificateAndKey:
+        secretName: my-source-secret
+        certificate: public.crt
+        key: private.key
+    config:
+      max.poll.records: 100
+      receive.buffer.bytes: 32768
+  producer:
+    bootstrapServers: my-target-cluster-kafka-bootstrap:9092
+    abortOnSendFailure: false <8>
+    config: <9>
+      compression.type: gzip
+      batch.size: 8192
+  whitelist: "my-topic|other-topic" <10>
+  resources: <11>
+    requests:
+      cpu: "8"
+      memory: 64Gi
+    limits:
+      cpu: "12"
+      memory: 128Gi
+  logging: <12>
+    type: inline
+    loggers:
+      _logger.name_: "INFO"
+  readinessProbe: <13>
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  livenessProbe:
+    initialDelaySeconds: 15
+    timeoutSeconds: 5
+  metrics: <14>
+    lowercaseOutputName: true
+    rules:
+      - pattern: "kafka.server<type=(.+), name=(.+)PerSec\\w*><>Count"
+        name: "kafka_server_$1_$2_total"
+      - pattern: "kafka.server<type=(.+), name=(.+)PerSec\\w*,
+        topic=(.+)><>Count"
+        name: "kafka_server_$1_$2_total"
+        labels:
+          topic: "$3"
+  jvmOptions: <15>
+    "-Xmx": "8g"
+    "-Xms": "8g"
+  image: my-org/my-image:latest <16>
+  template: <17>
+      pod:
+        affinity:
+          podAntiAffinity:
+            requiredDuringSchedulingIgnoredDuringExecution:
+              - labelSelector:
+                  matchExpressions:
+                    - key: application
+                      operator: In
+                      values:
+                        - postgresql
+                        - mongodb
+                topologyKey: "kubernetes.io/hostname"
+----
++
+<1> The number of replica nodes.
+<2> Bootstrap servers for consumer and producer.
+<3> Group ID for the consumer.
+<4> The number of consumer streams.
+<5> The offset auto-commit interval in milliseconds.
+<6> TLS encryption with key names under which TLS certificates are stored in X.509 format for consumer or producer.
+<7> Authentication for consumer or producer, using the TLS mechanism, as shown here, or a SASL-based SCRAM-SHA-512 or PLAIN mechanism.
+<8> Acknowledge (`true`) or ignore (`false`) a send failure for a message.
+<9> Kafka configuration options for consumer and producer.
```

Comment:
The (9) is now only at producer options. Should we add it to the consumer options as well to match the description?

Reply:
I've moved the number up to the consumer so that it flags the first time we see the config property.